src/components/Modal/ExtensionStatusModal.tsx

```
@@ -0,0 +1,300 @@
+                            onClick={onClose}
+                        >
+                            Close
+                        </button>
+                        {data?.allExtensionRequests &&
+                            data.allExtensionRequests.some(
+                                (request) => request.status === 'PENDING'
+                            ) === false && (
+                                <button
+                                    className={styles.extensionRequestButton}
+                                    onClick={handleOpenRequestForm}
+                                >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)